docs/modules/ROOT/pages/usage/server.adoc

```diff
@@ -49,7 +49,7 @@ user             16060   0.0  0.0  5078568   2264   ??  S     7:54AM   0:00.00 r
 user             16337   0.0  0.0  5331560   2396   ??  S    23:51PM   0:00.00 rubocop --server /Users/user/src/github.com/rubocop/rubocop-rails
 ```
 
-When you `bundle update` and run `rubocop`, the server process will restart automatically.
+When you running `bundle update` or updating a local config file (e.g., .rubocop.yml), and then running `rubocop`, the server process will automatically restart.
```

Comment:
```suggestion
When you run `bundle update` or update the root config file (e.g., `.rubocop.yml`), and then run `rubocop`, the server process will automatically restart.
```

Reply:
Since it could be located not only in the project root but also in the user's home directory, that part has been left as is, and the rest has been updated.